code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -15,7 +15,7 @@
 	coldmod = 6   // = 3x cold damage
 	heatmod = 0.5 // = 1/4x heat damage
 	burnmod = 0.5 // = 1/2x generic burn damage
-	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC| MIRROR_PRIDE | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT

```

Comment:
uh why are you swapping the order here?

Reply:
cause I poorly copy pasted them, fixing.